spec/openapi.yaml

```diff
@@ -1,50 +1,245 @@
+        required: true
+      responses:
+        200:
+          description: API key response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ApiKeyResponse'
+              example: |
+                {
+                  'apiVersion': '1.0',
+                  'data': {
+                    'apikey': 'yourapikey',
+                    'email': 'your@email.com'
+                  }
+                  'status': 'ok',
+                  'status_code': 200
+                }
+        401:
+          $ref: '#/components/responses/Unauthorized'
+        400:
+          $ref: '#/components/responses/BadRequest'
+  /search:
+    get:
+      summary: Search
+      description: Searches for resources containing a given keyword. For example, to search for any resource containing the term `binary trees`, make a `GET` request to `search?q=binary trees`
+      tags: 
+        - Search
+      parameters:
+        - in: query
+          name: q
+          description: Query parameter
+          schema:
+            type: string
+          required: true
+      responses:
+        200:
+          description: Search response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/SearchResponse'
+              example: |
+                {
+                  'apiVersion': '1.0',
+                  'data': [
+                      {
+                          'category': 'Tutorials',
+                          'created_at': '2019-05-25 18:52:15',
+                          'downvotes': 0,
+                          'id': 953,
+                          'languages': [
+                              'C'
+                          ],
+                          'last_updated': ',
+                          'name': 'C Tutorial',
+                          'notes': 'Learn C with our popular C tutorial, which will take you from the very basics of C all the way through sophisticated topics like binary trees and data structures.',
+                          'paid': false,
+                          'times_clicked': 0,
+                          'upvotes': 0,
+                          'url': 'https://www.cprogramming.com/tutorial/c-tutorial.html'
+                      }
+                  ],
+                  'number_of_pages': 1,
+                  'page': 0,
+                  'records_per_page': 20,
+                  'status': 'ok',
+                  'status_code': 200,
+                  'total_count': 1
+                }
+        404:
+          $ref: '#/components/responses/NotFound'
   /categories:
     get:
-      description: Gets a list of all available resource categories, such as books,
-        tutorials, and coding challenges.
+      summary: List Categories
+      description: Gets a list of all available resource categories, such as books, tutorials, and coding challenges.
```

Comment:
I'm not sure why, but this isn't showing up for me. See screenshot. Is this just happening on my machine?

![Screen Shot 2019-07-25 at 11 03 46 PM](https://user-images.githubusercontent.com/2008701/61925132-7ce23000-af30-11e9-8425-bdf2a2df4b6f.png)

Reply:
Weird. I'm seeing the same thing also but not sure why it isn't displaying. Will troubleshoot.

Reply:
Doh! It was missing its tag. All fixed now!